src/cc2olx/models.py

```diff
@@ -282,6 +286,16 @@ def get_resource_content(self, identifier):
                     )
                 )
                 return "html", {"html": html}
+            elif "web_resources" not in str(res_filename):
+                # Keep track of this file and later copy into static dir
```

Comment:
we can elaborate on this comment as to why it needs to be done in this way.

Reply:
Comment updated.